syncing latest head from Parse

premnirmal opened the pull request.

richardjrossiii committed:
Decouple object subclassing.

This fully decouples object subclassing from the ParseObject class, bringing it up to parity with our other native platforms.

daisuke-nomura committed:
replace deprecated

richardjrossiii committed:
Add configurable serverURL to Parse.Configuration.

Moves `server` from ParseObject to ParseRestCommand, and allows it to be
changed via `Builder.server("myUrl")`.

richardjrossiii committed:
Merge pull request #330 from ParsePlatform/richardross.configuration.serverurl

Add configurable serverURL to Parse.Configuration.

richardjrossiii committed:
Remove ObjectSubclassInfo class, fix unit test.

wangmengyan95 committed:
Update comment for ParseFile.cancel()

richardjrossiii committed:
Merge pull request #245 from ParsePlatform/richardross.subclassing.decouple

Decouple object subclassing.

wangmengyan95 committed:
Merge pull request #339 from ParsePlatform/wangmengyan.update_ParseFile_cancel_comment

Update comment for ParseFile.cancel()

grantland committed:
Update bolts-tasks to 1.4.0

grantland committed:
Merge pull request #310 from daisuke-nomura/master

Replace deprecated

wangmengyan95 committed:
Fix default parse server url

grantland committed:
Merge pull request #349 from ParsePlatform/grantland.bolts

Update bolts-tasks to 1.4.0

wangmengyan95 committed:
Avoid add unnecessary ParseDecompressInterceptor

wangmengyan95 committed:
Merge pull request #358 from ParsePlatform/wangmengyan.avoid_add_unnecessary_ParseDecompressInterceptor

Avoid add unnecessary ParseDecompressInterceptor

wangmengyan95 committed:
Remove Content-Encoding header when we auto unzip the response steram

wangmengyan95 committed:
Merge pull request #359 from ParsePlatform/wangmengyan.remove_content_encoding_header_when_auto_unzip_response

Remove Content-Encoding header when we auto unzip the response steram

wangmengyan95 committed:
Merge pull request #356 from ParsePlatform/wangmengyan.fix_server_url_bug

Fix default parse server url

wangmengyan95 committed:
Make eventuallyQueue initialization not rely on ParsePlugins

wangmengyan95 committed:
Merge pull request #361 from ParsePlatform/wangmengyan.make_initialize_eventuallyQueue_not_reply_on_ParsePlugins

Make eventuallyQueue initialization not rely on ParsePlugins

grantland committed:
:zap: Release 1.13.0 :zap:

grantland committed:
:tada: Prepare for next development iteration :confetti_ball:

Allsimon committed:
Add proguard consumer file

grantland committed:
Merge pull request #414 from Allsimon/master

Fix #343: Add proguard consumer file

anatolydwnld commented:
If you hold off for a couple of days on this, you will be able to remove this all together.

premnirmal commented:
ok, thanks @anatolydwnld

rogerhu committed:
Update for OkHttp3

rogerhu committed:
Update for OkHttp3

rogerhu committed:
For open source Parse with /parse mounts, this fix helps avoid the problem of remembering to add (#436)

the trailing slash.

Commit:
Removed file size check, Parse Android SDK

grantland committed:
Merge pull request #462 from kristoprifti/master

Removed file size check, Parse Android SDK

egk35 committed:
Don't override default SSL Socket Factory

This SSL Socket Factory is causing a javax.net.ssl.SSLPeerUnverifiedException exception when integrating an app with both Parse and Account Kit (https://developers.facebook.com/products/account-kit). Is there a reason why we're using the SSLCertificateSocketFactory? I think the default one should be fine.

grantland committed:
Merge pull request #465 from egk35/patch-2

Don't override default SSL Socket Factory

grantland committed:
Merge pull request #435 from rogerhu/okhttp3

Update for OkHttp3

grantland committed:
Merge pull request #468 from rogerhu/okhttp3

Update for OkHttp3

grantland committed:
Update Gradle

grantland committed:
:zap: Release 1.13.1 :zap:

grantland committed:
:tada: Prepare for next development iteration :confetti_ball:

Commit:
fix bug in ParseOkHttpClient that can cause errors when saving ParseFile

grantland committed:
Merge pull request #542 from zhoul-HS/master

fix bug in ParseOkHttpClient that can cause errors when saving ParseFile

JulienGenoud committed:
Update README.md

update links

grantland committed:
Merge pull request #584 from JulienGenoud/patch-1

Update README.md
